cmd/zb/perf.go

```diff
@@ -198,7 +200,17 @@ func updateStats(summary *statsSummary, record statsRecord) {
 	summary.latencies = append(summary.latencies, record.latency)
 }
 
-func printStats(requests int, summary *statsSummary) {
+type cicdTestSummary struct {
```

Comment:
I wouldn't use 'cicd' in types or variable names. This is an assumption on how the user would use the feature rather than what the feature does. Why not use something like 'jsonTestSummary'?

Reply:
"ci-cd" is specific to the ci-cd we are using and only outputs "requests per sec" in the cicdSummary format.
json format instead would have everything in JSON format, which will be separate.